doc/developer_guide/psyir.rst

```diff
@@ -572,9 +556,8 @@ elements of an array variable. It keeps a reference to a Symbol which
 will be stored in a symbol table. The indices used to access the array
 element(s) are represented by the children of the node. The
 ``ArrayReference`` Node inherits from both the ``Reference`` and
-``ArrayMixin`` classes.  See the full API in the `ArrayReference
-reference guide <https://psyclone-ref.readthedocs.io/en/latest/
-autogenerated/psyclone.psyir.nodes.html#psyclone.psyir.nodes.ArrayReference>`_.
+``ArrayMixin`` classes.  See the full API in the :ref_guide:`ArrayReference
+reference guide psyclone.psyir.nodes.html#psyclone.psyir.nodes.ArrayReference`.
 
 Directive
```

Comment:
There is a new link for the Directive node that was introduced by the merge and needs to be converted to `:ref_guide:` format

Reply:
Thanks for spotting - fixed.